core/platform/bulk_email/mailchimp_bulk_email_services.py

```diff
@@ -173,8 +174,12 @@ def add_or_update_user_status(
             identify the user in the mailchimp DB.
         can_receive_email_updates: bool. Whether they want to be subscribed to
             the bulk email list or not.
-        merge_fields: dict. Additional 'merge fields' used for Android signups
-            currently, which can be expanded in the future.
+        merge_fields: dict. Additional 'merge fields' used by mailchimp for
+            adding extra information for each user. The format is
+            { 'KEY': value } where the key is defined in the mailchimp
+            dashboard .
```

Comment:
Drop space before period, or drop the period and move "(reference: " to this line.

Reply:
Done